server/src/main/java/io/druid/segment/realtime/appenderator/AppenderatorImpl.java

```diff
@@ -283,6 +283,18 @@ int getRowsInMemory()
     return rowsCurrentlyInMemory.get();
   }
 
+  @VisibleForTesting
+  long getRowSizeInMemory(SegmentIdentifier identifier)
```

Comment:
Would you rename this method to a more intuitive one? Looks like it returns a row size.

Reply:
Probably `getBytesInMemory`?

Reply:
done